src/test/resources/net/sf/jabref/importer/fileformat/InspecSilverPlatterImporterTest.txt

```diff
@@ -1,30 +0,0 @@
-Record INSPEC
```

Comment:
Personal discussion: test case covered by other files. Therefore, OK to remove. @tschechlovdev